src/core/utility/file/filePath/FilePath.cpp

```diff
@@ -63,7 +53,7 @@ const FilePath& FilePath::EmptyFilePath() {
   return sEmptyFilePath;
 }
 
-boost::filesystem::path FilePath::getPath() const {
+fs::path FilePath::getPath() const {
   return *(m_path.get());
```

Comment:
```suggestion
  return *m_path;
```

Reply:
Operator * is already overloaded.
I agree.